src/aat/java/uk/gov/hmcts/ccd/fta/steps/BackEndFunctionalTestScenarioPlayer.java

```diff
@@ -38,13 +28,22 @@
 import uk.gov.hmcts.ccd.fta.util.EnvUtils;
 import uk.gov.hmcts.ccd.fta.util.JsonUtils;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
 
 @SuppressWarnings({ "LocalVariableName" })
 public class BackEndFunctionalTestScenarioPlayer implements BackEndFunctionalTestAutomationDSL {
 
     private final String BE_FTA_FILE_JURISDICTION1 = "src/aat/resources/CCD_BEFTA_JURISDICTION1.xlsx";
     private final String BE_FTA_FILE_JURISDICTION2 = "src/aat/resources/CCD_BEFTA_JURISDICTION2.xlsx";
     private final String BE_FTA_FILE_JURISDICTION3 = "src/aat/resources/CCD_BEFTA_JURISDICTION3.xlsx";
+    private final String BE_FTA_FILE_JURISDICTION4 = "src/aat/resources/CCD_BEFTA_JURISDICTION4.xlsx";
```

Comment:
Please add the case type in the file of jurisdiction 1. All jurisdictions have a respective excel file. An additional case type under jurisdiction1 should be defined inside CCD_BEFTA_JURISDICTION1.xlsx.

Reply:
Added into CCD_BEFTA_JURISDICTION1.xlsx.